src/lib/constants/resourceTypes.ts

```diff
@@ -53,7 +54,7 @@ export const PARAGRAPH_RESOURCE_TYPES = {
   PROCESS_LIST: 'paragraph--process',
   REACT_WIDGET: 'paragraph--react_widget',
   RICH_TEXT_CHAR_LIMIT_1000: 'paragraph--rich_text_char_limit_1000',
-  STAFF_PROFILE: 'paragraph--staff_profile',
+  STAFF_PROFILE_PARAGRAPH: 'paragraph--staff_profile',
```

Comment:
This isn't needed.

Reply:
There is a paragraph of staff profile, but we aren't handling it yet. I'll remove it for now.

Reply:
Er, specifically, I meant the rename isn't needed. `PARAGRAPH_RESOURCE_TYPES.STAFF_PROFILE` isn't likely to be conflicting with `RESOURCE_TYPES.STAFF_PROFILE` unless we're deconstructing both objects in the same scope for some reason, thus causing a naming conflict. 🤷